README.md

```diff
@@ -1,38 +1,151 @@
+  
+  - **In Kotlin**
+  
+  ```kotlin
+    //Create an Object of mutation by passing in the build parameters according to the scehma.
+    val mutation = UpdateCurrentTaskMutation.builder().id(id).title(title).version(version).build()
+    
+    //Create a client which is an object of ApolloCall on which call would be made.
+     val client = Utils.getApolloClient(context)?.mutate(mutation)?.refetchQueries(apolloQueryWatcher?.operation()?.name())
+     
+    //Create a callback object of type ApolloCall.Callback<UpdateCurrentTaskMutation.Data>
+     val callback = object : ApolloCall.Callback<UpdateCurrentTaskMutation.Data>() {
+            override fun onFailure(e: ApolloException) {
+                e.printStackTrace()
+            }
 
-1. Make a mutation by going offline.
-2. Then, come online and click on "FromDb" button and you will see the results.
-3. Also, refresh the application to see the updated cache.
+            override fun onResponse(response: Response<UpdateCurrentTaskMutation.Data>) {
+              //Perform UI Bindings here.
+            }
+        }
+        
+     /*Call the enqueue function on the instance of APolloClient and pass in two parameters here:
+       1. mutation object typecasted as mutation as 
+          com.apollographql.apollo.api.Mutation<Operation.Data, Any, Operation.Variables>                 
+       2. callback object typecasted as callback as ApolloCall.Callback<Any>
+     */  
+     Utils.getApolloClient(context)?.enqueue(
+            mutation as com.apollographql.apollo.api.Mutation<Operation.Data, Any, Operation.Variables>,
+            callback as ApolloCall.Callback<Any>
+        )
+```
 
+  - **In Java**
+  
+  ```java
+    //Create an Object of mutation by passing in the build parameters according to the scehma.
+    Mutation mutation = UpdateCurrentTaskMutation.builder().id(id).title(title).version(version).build();
+    
+    //Create a client which is an object of ApolloCall on which call would be made.
+    ApolloMutationCall<UpdateCurrentTaskMutation.Data> client = Utils.INSTANCE.getApolloClient(context).mutate(mutation)
+                .refetchQueries(apolloQueryWatcher.operation().name());  
+                
+    //Create a callback object of type ApolloCall.Callback<UpdateCurrentTaskMutation.Data>
+     ApolloCall.Callback callback = new ApolloCall.Callback<UpdateCurrentTaskMutation.Data>(){
+            @Override
+            public void onResponse(@NotNull Response<UpdateCurrentTaskMutation.Data> response) {
+            //Perform UI Bindings here.
+            }
+
+            @Override
+            public void onFailure(@NotNull ApolloException e) {
+            }
+        };
+        
+     /* Call the enqueue function present in the file ExtensionKt which takes in 3 parameters:
+        1. apollo client 
+        2. mutation object              
+        3. callback object
+      */
+        ExtensionKt.enqueue(Utils.INSTANCE.getApolloClient(context),
```

Comment:
@Lavanyagaur22 Let's rename the `ExtensionsKi` file to `Offix`

Reply:
Also, not always the users will be getting their ApolloClient via Utils, so no need to include that here.
The example can simply say : 

```
Offix.enqueu(apolloClient,mutation,callback)
```

Reply:
Also, @Lavanyagaur22 can you please add `@JVMStatic` annotation on the `getApolloClient()` method?
That way, we can access it directly in java instead of adding `INSTANCE` in between.